src/config/pulse-ext.php

```diff
@@ -4,8 +4,11 @@
 
     'record_interval' => 60,
 
-    'queue_list' => 'pulse_queues',
+    // Must be a key from the queue:monitor output pointing to an integer
+    'queue_status' => ['pending', 'delayed', 'reserved'],
```

Comment:
The comment seems to be misleading here. Also I don't see a need for this to be configurable. The different "sub"-queues are always called like this in Laravel.